index.html

```diff
@@ -113,15 +113,81 @@ <h2 class="courses-main-copy">Courses</h2>
           </div>
         </div>
 
-        <div class="courses-button">
-          <div class="courses-button-copy"><p>すべてのコースを見る</p></div>
+        <div class="button" id="courses-button">
```

Comment:
```suggestion
        <div class="button courses-button">
```
idを使うと詳細度が上がってしまうので、別のスタイルを当てたい場合はこのように2つ目のクラスを指定するのが良いかなと思います。
ただ現状 `couses-button` に対してはスタイルが当たっていないと思うので、そもそもなくしてしまっても良いかもしれません。